util/check-format.pl

```diff
@@ -881,7 +881,7 @@ sub check_nested_nonblock_indents {
         $if_maybe_terminated = 0;
         if (my ($head, $before, $tail) = m/^([\s@]*([^{}]*)\})[\s@]*(.*)$/) { # leading closing '}', but possibly
                                                                               # with non-whitespace non-'{' before
-            report("code after '}'") unless $tail eq "" || $tail =~ m/(else|while|OSSL_TRACE_END)(\W|$)/;
+            report("code after '}'") unless $tail eq "" || $tail =~ m/(else|while|$)/;
```

Comment:
Not sure, but should that not be 
```suggestion
            report("code after '}'") unless $tail eq "" || $tail =~ m/(else|while)(\W|$)/;
```
to ensure that there's a word boundary after while and else?

Reply:
LGTM.

Reply:
Did you mean to change this or not? It seems to me that e.g. `elsewhere` now matches and previously it would not.

Reply:
oops. didn't write out the buffer...